debian/changelog

```diff
@@ -1,3 +1,11 @@
+hidpi-daemon (18.04.7) bionic; urgency=low
+
+  * Update hidpidaemon2.py file to set gi require version
+  * Add newer models with HiDPI displays
+  * Update Maintainer
+
+ -- Aaron Honeycutt <aaron@system76.com>  Tue, 02 Jun 2020 09:18:29 -0600
+
 hidpi-daemon (18.04.6~~alpha) bionic; urgency=low
 
   * Daily WIP for 18.04.6
```

Comment:
You need to move your updates below here and remove 18.04.7 version. There is a script that updates this package which I will run when this PR is merged.

Reply:
I believe commit 50a9916 will fix that.